theano/scalar/basic.py

```diff
@@ -851,6 +851,8 @@ class ScalarOp(Op):
     nin = -1
     nout = 1
 
+    __props__ = ("output_types_preference", "name")
```

Comment:
To be clear, all the sub classes that has `__init__` should also be having `__props__` right ?